lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -222,6 +222,50 @@ SmallVector<unsigned> getWarpOrder(DistributedEncodingTrait layout,
   return toLinearEncoding(layout, shape).getWarpOrder();
 }
 
+bool isDescendingOrder(triton::gpu::MemDescType type) {
```

Comment:
@lezcano @Jokeren could one of you take a look at this?

Reply:
I think there are a few things to discuss. Will drop comments below